src/components/Inputs.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+// to create an input form to insert new guest
+class Inputs extends React.Component {

```

Comment:
Try to find a more specific name for this component. `Inputs` tells me that it's going to be several inputs but nothing more. Here, you actually have a form to add a new booking for a guest, how about naming this component `AddBookingGuestForm` ?